src/main/java/mate/academy/dao/BookDaoImpl.java

```diff
@@ -0,0 +1,114 @@
+package mate.academy.dao;
+
+import java.math.BigDecimal;
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import mate.academy.exception.DataProcessingException;
+import mate.academy.lib.Dao;
+import mate.academy.model.Book;
+import mate.academy.model.ConnectionUtil;
+
+@Dao
+public class BookDaoImpl implements BookDao {
+    private static final String CREATE_REQUEST = "INSERT INTO books (title, price) VALUES (?, ?)";
```

Comment:
CREATE_QUERY or INSERT_QUERY are better because it says about sql query

Minor comment